server/config.yml

```diff
@@ -17,7 +17,7 @@ server:
         ca-certificate:
 
 storage:
-    data-directory: "data"
+    database-directory: "data"
```

Comment:
Hmm actually i think 'data' directory is better - aligns with users will think of when they see "data" (their own data), instead of what we consider data (anything we store)